tests/storage/test_roommember.py

```diff
@@ -197,9 +192,4 @@ def test_can_rerun_update(self):
         self.store.db_pool.updates._all_done = False
```

Comment:
I did have one concern here. Some of these explicitly reset the `_all_done` flag, and others don't. I haven't made the situation worse---apart from an extra stack frame the behaviour should be identical. But I wondered if it was was more consistent to reset the flag in the helper function.

Reply:
:man_shrugging: